From 141eafa531c6c09a06efe6a694251a1eea84908d Mon Sep 17 00:00:00 2001
From: Thomas A Caswell <tcaswell@bnl.gov>
Date: Sun, 17 Mar 2019 16:57:16 -0400
Subject: [PATCH] TST: bump skip version on attrs test

We are exercising a bug in hdf5, projected to be fixed in 1.10.6 and
1.12.0

closes #1180
---
 h5py/tests/old/test_attrs.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/h5py/tests/old/test_attrs.py b/h5py/tests/old/test_attrs.py
index d2e0ab59..46b20e8e 100644
--- a/h5py/tests/old/test_attrs.py
+++ b/h5py/tests/old/test_attrs.py
@@ -178,7 +178,7 @@ def fill_attrs(self, track_order):
             attrs[str(i)] = i
         return attrs
 
-    @ut.skipUnless(h5py.version.hdf5_version_tuple >= (1, 10, 5), 'HDF5 1.10.5 required')
+    @ut.skipUnless(h5py.version.hdf5_version_tuple >= (1, 10, 6), 'HDF5 1.10.6 required')
     # https://forum.hdfgroup.org/t/bug-h5arename-fails-unexpectedly/4881
     def test_track_order(self):
         attrs = self.fill_attrs(track_order=True)  # creation order

